services/api/src/utils/mailer.js

```diff
@@ -1,15 +1,107 @@
 const postmark = require('postmark');
-
 const marked = require('marked');
 const Mustache = require('mustache');
+const { memoize } = require('lodash');
+const fm = require('front-matter');
 const fs = require('fs').promises;
 const path = require('path');
 
 const config = require('@bedrockio/config');
 const { logger } = require('@bedrockio/instrumentation');
 
-const POSTMARK_FROM = config.get('POSTMARK_FROM');
-const ENV_NAME = config.get('ENV_NAME');
+const ENV = config.getAll();
+
+const { ENV_NAME, POSTMARK_FROM, POSTMARK_API_KEY, POSTMARK_SEND_DEV } = ENV;
+
+async function sendTemplatedMail({ template, layout = 'layout.html', to, ...options }) {
+  const textTemplate = template.replace(/\.md$/, '.txt');
+  let [
+    { body: layoutBody, meta: layoutMeta },
+    { body: templateBody, meta: templateMeta },
+    { body: textBody, meta: textMeta },
+  ] = await Promise.all([fetchTemplate(layout), fetchTemplate(template), fetchTemplate(textTemplate)]);
+
+  if (!templateBody && !textBody) {
+    logger.error(`Could not load template ${template}.`);
+    return;
+  }
+
+  options = {
```

Comment:
could we call this something else than `options`, redeclaring vars makes me confused

Reply:
Maybe call it `vars` instead to drive home that the variables are interpolated?